Content.Server/Chemistry/ReagentEffects/PlantMetabolism/PlantPhalanximine.cs

```diff
@@ -0,0 +1,36 @@
+using Content.Server.Botany.Components;
+using Content.Server.Botany.Systems;
+using Content.Shared.Chemistry.Reagent;
+using JetBrains.Annotations;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Random;
+using Robust.Shared.Toolshed.TypeParsers;
+
+namespace Content.Server.Chemistry.ReagentEffects.PlantMetabolism
+{
+    [UsedImplicitly]
+    [DataDefinition]
+    public sealed partial class PlantPhalanximine : ReagentEffect
+    {
+        public override void Effect(ReagentEffectArgs args)
+        {
+            if (!args.EntityManager.TryGetComponent(args.SolutionEntity, out PlantHolderComponent? plantHolderComp)
+                                    || plantHolderComp.Seed == null || plantHolderComp.Dead ||
+                                    plantHolderComp.Seed.Immutable)
+                return;
+
+
+            var plantHolder = args.EntityManager.System<PlantHolderSystem>();
+            var random = IoCManager.Resolve<IRobustRandom>();
+            if (args.Source == null) return;
+            var source = args.Source;
+            //Need at least 5u, had to do it like this because plant metabolism is weird
+            if (source.TryGetReagent("Phalanximine", out var quantity) && quantity >= 4.00)
```

Comment:
Reagent quantity is available from the `ReagentEffectArgs`

Reply:
If you're talking about `ReagentEffectArgs.Quantity`, it seems to return only the quantity being processed during that metabolism tick, not the overall amount of that reagent within the solution, so I don't think I can use it for that.

Reply:
Could you check if reagent conditions are appropriate? e.g.
```
        conditions:
        - !type:ReagentThreshold
          min: 5
```

Reply:
I believe I tried it once and it didn't work (which led to this less than ideal solution) but I'll try again just to be sure.

Reply:
Turns out I was trying to do it wrong, whoooops. Should be ready to ship now, thanks for getting me to look at it again.